src/System.Diagnostics.Process/src/System/Diagnostics/Process.cs

```diff
@@ -310,6 +310,7 @@ public long NonpagedSystemMemorySize64
             }
         }
```

Comment:
you are right, I did miss a few. I was doing my comparison against bin\layout\ref\netstandard1.7 which for some reason doesn't have System.Diagnostics.Process which is why I missed it in the first pass. This time I used bin\layout\ref\netcoreapp1.1 and got all of those missing ones.